rollup.config.js

```diff
@@ -1,146 +1,59 @@
-/* eslint-env es6 */
+import resolve from '@rollup/plugin-node-resolve';
+import cjs from "rollup-plugin-cjs-es";
```

Comment:
What's the difference between this and the `rollup-plugin-commonjs` that was used before?

Reply:
`cjs-es` produced much better code at one point. i re-tested it just now and they're *mostly* equivalent (i still prefer the intermediate names it produces better), and `commonjs` still injects some helpers that don't seem to be necessary, so the minified build ends up a 800b larger.